### ISSUE TITLE
Map viewer needs to allow scroll wheel to scroll page when mouse within it

### ISSUE THREAD

**Issue by diedrick** [bplmaps/argomaps#49]
For map pages:
https://argo-staging.netlify.app/maps/commonwealth:hx11z0241/

The map viewer itself is preventing scroll. I've tried to workaround Tify's implementation since it has almost no documentation or support to work with Openseadragon, but I can't find a way to implement that. Got any fixes in your experience with Tify?

In the future we might want to move away from Tify to a viewer that's more supported, and that has extra features like annotations, so any recommendations there would be worthwhile.

![image](https://user-images.githubusercontent.com/9010288/217644447-0a40de33-8ebf-4311-95a3-4fcf7e8e335d.png)

**Comment by almontg** [comment]
Echoing Michael here — Garrett is there a compelling reason to use Tify as opposed to openseadragon or mirador? I'd really like to get a solution to the annotations issue and it's something I know mirador supports, do using that as a viewer would be a relatively easy solution

**Comment by garrettdashnelson** [comment]
This is a known issue with OpenSeadragon, see https://github.com/openseadragon/openseadragon/issues/2151

It looks like it may get corrected by default in a future release of OSD, but meanwhile can be addressed by setting the `viewer` element (which is OSD inside of Tify) as such:

```
viewer.addHandler('canvas-scroll', function(event) {
    event.preventDefault = false;
});
```

I don't feel super strongly about what viewer we use, though I personally like Tify best. Its actual zoomer component is just OSD wrapped in Vue, and OSD is the default plugin environment for [Annotorious](https://recogito.github.io/annotorious/), which is what I'd imagine we would spec out for an annotations system anyhow.

**Comment by diedrick** [comment]
That's the trick I tried to implement but I can't find where to put it, variations like this didn't work:
![image](https://user-images.githubusercontent.com/9010288/217871781-eaf28cad-f90f-43a1-8290-00580882d427.png)

Thoughts?

**Comment by garrettdashnelson** [comment]
Implemented this at [5069155](https://github.com/bplmaps/argomaps/commit/50691553a93d658e1374d6ede4d021548d88b1d6).
